static/styles/components.css

```diff
@@ -144,3 +144,49 @@ select.bootstrap-focus-style {
         outline-offset: -2px;
     }
 }
+
+.progress {
+    overflow: hidden;
+    background-color: hsl(0, 0, 97);
+    background-image: linear-gradient(
+        to bottom,
+        hsl(0, 0%, 96%),
+        hsl(0, 0%, 98%)
+    );
+    background-repeat: repeat-x;
+    box-shadow: inset 0 1px 2px hsla(0, 0%, 0%, 0.1);
+    border-radius: 4px;
+}
+
+.progress .bar {
+    width: 0%;
+    height: 100%;
+    color: hsl(0, 0%, 100%);
+    float: left;
+    font-size: 12px;
+    text-align: center;
+    text-shadow: 0 -1px 0 hsla(0, 0%, 0%, 0.25);
+    box-shadow: inset 0 -1px 0 hsla(0, 0%, 0%, 0.15);
+    box-sizing: border-box;
+    transition: width 0.6s ease;
+}
+
+.progress .bar-danger {
+    background-color: hsl(2, 68, 58);
```

Comment:
This should be `hsl(2, 68%, 58%)` I think.

Reply:
And this similar change would be required for other `background-color` properties as well.